src/OSPSuite.Presentation/Presenters/Main/IMainViewPresenter.cs

```diff
@@ -13,9 +12,7 @@ public interface IMainViewPresenter :
       IListener<HeavyWorkFinishedEvent>,
       IListener<HeavyWorkStartedEvent>,
       IListener<RollBackStartedEvent>,
-      IListener<RollBackFinishedEvent>,
-      IListener<ReportCreationStartedEvent>,
```

Comment:
Those two events were implemented as abstract from the interface. Remove them. Detailled implementation in all apps